k8s/rabbitmq/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,60 @@
 actions:
-- name: dummy test
+- name: RabbitMQ test
   bashTest:
     script: |-
-      exit 0
+      SERVICE="${APP_INSTANCE_NAME}-rabbitmq-svc.${NAMESPACE}.svc.cluster.local"
+      PORT=15672
+      USERNAME=''
+      PASSWORD=''
+      FAILED=0
+      DEBUG=1
+
+      # getting RabbitMQ username
+      echo "Getting RabbitMQ username."
+      USERNAME=$(kubectl get sts -oyaml \
+        --namespace "${NAMESPACE}" "${APP_INSTANCE_NAME}-rabbitmq" | \
+        grep -A1 name:.RABBITMQ_DEFAULT_USER | sed -n '/value: /s/.* //p')
+
+      # getting RabbitMQ user's password
+      echo "Getting RabbitMQ ${USERNAME} password."
+      PASSWORD=$(kubectl get secret "${APP_INSTANCE_NAME}-rabbitmq-secret" \
+        --namespace "${NAMESPACE}" \
+        --output=jsonpath='{.data.rabbitmq-pass}' | base64 -d)
+
+      # test1 -- downloading RabbitMQ CLI tool
```

Comment:
Please move this to the separate tests. I know that tests below can't finish successfully without a CLI, but it should be in the seperated actions as the framework was designed for.

Also, we will catch that someone deleted CLI action thanks to CI pipeline.